arelle/plugin/validate/ESEF/resources/authority-validations.json

```diff
@@ -0,0 +1,181 @@
+        "",
+        "reportPackageMaxMB: max file size in MB followed by \" zipped\" or \" unzipped\", or unlimited",
+        "ixTargetUsage: ok, warning or error",
+        "extensionAbstractContexts: ok, warning or error",
+        "G3.4.2: effective or ineffective when not applied by authority",
+        "G3.4.6: effective or ineffective when not applied by authority",
+        "additionalMandatoryTags: qnames of tags additionally required by authority when prefixed schema is imported"
+    ],
+    "default": {
+        "comment": "default parameters where not specified by authority",
+        "reportPackageMaxMB": "unlimited",
+        "identiferScheme": "http://standards.iso.org/iso/17442",
+	    "ixTargetUsage": "warning",
+        "extensionAbstractContexts": "warning",
+		"G3.4.2": "effective",
+		"G3.4.6": "effective",
+	    "outdatedTaxonomyURLs": [
+		    "http://www.esma.europa.eu/taxonomy/2017-03-31/esef_cor.xsd",
+		    "https://www.esma.europa.eu/taxonomy/2017-03-31/esef_cor.xsd"
+		],
+		"effectiveTaxonomyURLs": [
+		    "http://www.esma.europa.eu/taxonomy/2019-03-27/esef_cor.xsd",
+		    "https://www.esma.europa.eu/taxonomy/2019-03-27/esef_cor.xsd",
+		    "http://www.esma.europa.eu/taxonomy/2020-03-16/esef_cor.xsd",
+		    "https://www.esma.europa.eu/taxonomy/2020-03-16/esef_cor.xsd",
+		    "http://www.esma.europa.eu/taxonomy/2021-03-24/esef_cor.xsd",
+		    "https://www.esma.europa.eu/taxonomy/2021-03-24/esef_cor.xsd"
+		]
+    },
+	"AT": {
+	    "name": "Austria",
+	    "reportPackageMaxMB": "100 zipped"
+	},
+	"BE": {
+	    "name": "Belgium",
+	    "reportPackageMaxMB": "50 zipped"
+	},
+	"BG": {
+	    "name": "Bulgaria"
+	},
+	"CY": {
+	    "name": "Cyprus"
+	},
+	"CZ": {
+	    "name": "Czechia"
+	},
+	"DE": {
+	    "name": "Germany"
+	},
+	"DK": {
+	    "name": "Denmark",
+	    "reportPackageMaxMB": "2000 unzipped",
+	    "ixTargetUsage": "ok"
+	},
+	"EE": {
+	    "name": "Estonia",
+	    "reportPackageMaxMB": "unlimited"
+	},
+	"EL": {
+	    "name": "Greece"
+	},
+	"ES": {
+	    "name": "Spain"
+	},
+	"FI": {
+	    "name": "Finland"
+	},
+	"FR": {
+	    "name": "France",
+	    "reportPackageMaxMB": "40 zipped"
+	},
+	"HR": {
+	    "name": "Croatia"
+	},
+	"HU": {
+	    "name": "Hungary"
+	},
+	"IE": {
+	    "name": "Ireland"
+	},
+	"IS": {
+	    "name": "Iceland"
+	},
+	"IT": {
+	    "name": "Italy"
+	},
+	"LI": {
+	    "name": "Liechtenstein"
+	},
+	"LT": {
+	    "name": "Lithuania"
+	},
+	"LU": {
+	    "name": "Luxembourg"
+	},
+	"LV": {
+	    "name": "Latvia"
+	},
+	"MT": {
+	    "name": "Malta",
+	    "reportPackageMaxMB": "25 zipped"
+	},
+	"NL": {
+	    "name": "Netherlands",
+	    "reportPackageMaxMB": "100 unzipped",
+	    "comment": "no suffix on report file names",
+	    "reportFileNamePattern": "{base}-{date}{report}.xhtml",
+	    "extensionElementsAnchoring": "include abstract",
+	    "dimension-default-arc-ELRs": "990000 only",
+	    "LC3": "Capital letters allowed in words"
```

Comment:
I dont see where these values are being used?

Reply:
Implemented NL options:
        "LC3CapitalsInWord: true means label \"NonControlling Interest\" matches name \"NoncontrollingInterest\"",
        "reportFileNamePattern: filename pattern where applied by authority, for message display",
        "reportFileNameRegex: regex to correspond, in same manner as filenameRegexes in Consts.py",
        "extensionElementsAnchoring: \"non-abstract\" or \"include abstract\"",